test/tests/builtin_type_not_subclassable.py

```diff
@@ -1,3 +1,19 @@
 # should_error
+import re
+
 class foo(slice):
    pass
+
+class C(type(xrange(1))):
```

Comment:
@kmod hmm You're right. than should I seperate test file or is there any good solution?